src/main/config/config.c

```diff
@@ -617,7 +617,7 @@ static void resetConf(void)
     masterConfig.rxConfig.spektrum_sat_bind = 5;
     masterConfig.escAndServoConfig.minthrottle = 1000;
     masterConfig.escAndServoConfig.maxthrottle = 2000;
-    masterConfig.motor_pwm_rate = 32000;
+    masterConfig.motor_pwm_rate = BRUSHED_MOTORS_PWM_RATE;
```

Comment:
may be better keep it since the BRUSHED_MOTORS_PWM_RATE is only 16000 and define in the same source file.